pkg/expression/helper.go

```diff
@@ -162,32 +158,5 @@ func getStmtTimestamp(ctx EvalContext) (time.Time, error) {
 		v := time.Unix(int64(val.(int)), 0)
 		failpoint.Return(v, nil)
 	})
-
-	if ctx != nil {
```

Comment:
These code has been moved to: https://github.com/pingcap/tidb/pull/52015/files#diff-f1e249804afb644119e933b3c10bd07625d920b322393f8b3be95d192d50d608R180